src/convertToObject.js

```diff
@@ -1,18 +1,36 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourseArr = sourceString.split('\n');
+
+  const result = sourseArr.reduce((acc, el) => {
+    const parts = el.split(':');
+    const key = parts[0].trim();
+
+    let value = parts.slice(1).toString().trim();
```

Comment:
4. [CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable.


BAD EXAMPLE:
```
const cssStyles = styles.map(style => style).filter(style => style.length).slice(0, 5)
```

GOOD EXAMPLE: 
```
const cssStyles = styles
 .map(style => style)
 .filter(style => style.length)
 .slice(0, 5)
```
It's also belongs to string methods, we need to make them readable too